src/popup/router/routes.ts

```diff
@@ -146,7 +150,11 @@ export const routes: WalletAppRouteConfig[] = [
           },
           {
             path: 'names',
-            component: AccountDetailsNames,
+            component: ProtocolSpecificView,
+            props: {
+              viewComponentName: PROTOCOL_VIEW_ACCOUNT_DETAILS_NAMES,
+              redirectIfNull: ROUTE_ACCOUNT_DETAILS,
```

Comment:
In my opinion this should be passed as the meta instead of prop because the meta can be typed. So in case of future refactoring it will be easier to do it by reading what TS is complaining about.

I know that I've made the initial change that was passing the `viewComponentName` value as a prop. But I consider this as a mistake that should be fixed in near future.